### ISSUE TITLE
Fields with addons are not correctly aligned

### ISSUE THREAD

**Issue by protmjones** [mdbootstrap/mdb-ui-kit#262]
Using the following HTML, the two inputs should appear on the same line, but do not. 

Tested on Firefox, Chrome and IE11.

It would appear that it is the javaScript that is causing the issue - if I remove the material.min.js script, I lose the focus effect, but the baselines line up

```
    <div class="container">
        <form>
        <div class="row">
            <div class="col-md-6">
                <div class="input-group">
                <span class="input-group-addon">@</span>
                <input type="text" class="form-control" placeholder="Username">
                </div>
            </div>
            <div class="col-md-6">
                <input type="text" class="form-control" placeholder="Another field">
            </div>
        </div>
        </form>
    </div>
```

**Comment by FezVrasta** [comment]
http://codepen.io/FezVrasta/pen/WbQaJx

I see them on the same line (with some padding problem that should be fixed).

May you provide a pen of your problem?

**Comment by protmjones** [comment]
http://www.evernote.com/shard/s439/sh/afa4a886-a420-40c0-8db1-21255f74ff2d/f884cb5f5850c8a207ec4374fee89160

They are on the same row, but the baselines do not line up. Is this the padding you speak of?

**Comment by FezVrasta** [comment]
yup it is, it must be fixed.

**Comment by FezVrasta** [comment]
this problem should be fixed, please reopen if not

**Comment by protmjones** [comment]
The fields are aligned now. However - the cool onFocus effect no longer seems to occur - is this related, or has this been removed?

**Comment by FezVrasta** [comment]
Sorry there was a bug with the script, I've fixed it. Try now.

**Comment by protmjones** [comment]
dropdowns do not line up with other fields

http://www.evernote.com/shard/s439/sh/99986c89-4709-40f6-9ea4-bd01d649ebb8/0c965846eb0f03c48cf34a829ca278a7

```
<div class="row">
    <div class="col-sm-5">
        <div class="form-group field-demo-test_date">
            <div class="input-group date">
                <span class='input-group-addon'><i class="glyphicon glyphicon-calendar"></i></span>
                <input type="text" id="field" class="form-control" name="field" value="">
            </div>
        </div>
    </div>
    <div class="col-md-6">
        <select class="form-control">
            <option value="volvo">Volvo</option>
            <option value="saab">Saab</option>
            <option value="mercedes">Mercedes</option>
            <option value="audi">Audi</option>
        </select>
    </div>
</div>
```

**Comment by ravikantn** [comment]
Fields with button addons do not align correctly. The input crosses over the button

```
<div class="input-group">
     <input placeholder="Image URL" type="text" id="imageUrl" class="form-control">
          <span class="input-group-btn">
        <button class="btn btn-primary " type="button">Insert</button>
    </span>
</div>
```

**Comment by rosskevin** [comment]
Resolved by #725 and released as [0.4.0](https://github.com/FezVrasta/bootstrap-material-design/releases/tag/0.4.0)
